data/json/monsters/zed-winged.json

```
@@ -142,5 +143,49 @@
       "SWARMS",
       "FLIES"
     ]
+  },
+  {
+    "type": "MONSTER",
+    "id": "mon_spawn_raptor_shady",
+    "name": { "str_sp": "shady flesh-raptor" },
+    "copy-from": "mon_spawn_raptor",
+    "color": "light_gray",
+    "description": "An uncanny shadow envelops this creature.  You can make out the outline of three flapping wings connected to a jagged pointed object.",
+    "upgrades": {  },
+    "extend": { "flags": [ "NIGHT_INVISIBILITY" ] },
+    "vision_day": 5,
```

```suggestion
    "vision_day": 10,
```
It has to be able to see at least a little by day.